File: KingdomCodeTool/wwwroot/Download/202503250844/Service/Implement/CLSYeuCau_ThayDoiService.cs
namespace Service_eHospital_DongNai_A.Implement
{
    public class CLSYeuCau_ThayDoiService : BaseService<CLSYeuCau_ThayDoi, ICLSYeuCau_ThayDoiRepository>
    , ICLSYeuCau_ThayDoiService
    {
    private readonly ICLSYeuCau_ThayDoiRepository _CLSYeuCau_ThayDoiRepository;
    public CLSYeuCau_ThayDoiService(ICLSYeuCau_ThayDoiRepository CLSYeuCau_ThayDoiRepository) : base(CLSYeuCau_ThayDoiRepository)
    {
    _CLSYeuCau_ThayDoiRepository = CLSYeuCau_ThayDoiRepository;
    }
    }
    }

File: KingdomCodeTool/wwwroot/Download/202503250844/Angular/shared/CLSYeuCau_ThayDoi.service.ts
import { Injectable } from '@angular/core';
import { HttpClient, HttpParams } from '@angular/common/http';
import { environment } from 'src/environments/environment';
import { CLSYeuCau_ThayDoi } from './CLSYeuCau_ThayDoi.model';
import { BaseService } from './Base.service';
@Injectable({
    providedIn: 'root'
})
export class CLSYeuCau_ThayDoiService extends BaseService{
    DisplayColumns001: string[] = ['Save', 'STT', 'CLSYeuCau_ThayDoi_Id', 'CLSYeuCau_Id', 'PhongBan_Id', 'PhongBan_ThayDoi_Id', 'LanThayDoi', 'NgayThayDoi', 'ThoiGianThayDoi', 'NguoiThayDoi_Id', 'NgayTao', 'NguoiTao_Id', 'NgayCapNhat', 'NguoiCapNhat_Id'];

    List: CLSYeuCau_ThayDoi[] | undefined;
    ListFilter: CLSYeuCau_ThayDoi[] | undefined;
    FormData!: CLSYeuCau_ThayDoi;
    constructor(public httpClient: HttpClient) {
        super(httpClient);
        this.Controller = "CLSYeuCau_ThayDoi";
    }
}

